File: Week01/Sample01.java
import java.util.Scanner;

public class Sample01{
    public static void main(String [] args){
        // get the target value and save it in the variable, temp
        int temp = Integer.valueOf(args[0]);
        Scanner helper = new Scanner(System.in);

        while(true){
            System.out.print("Guess : ");
            int guess = helper.nextInt();

            if(temp<guess){
                System.out.println("DOWN");
            }else if(temp>guess){
                System.out.println("UP");
            }else {
                System.out.println("BINGO");
                break;
            }
        }
        System.out.println("GAME IS OVER");

    }
}

File: Week01/Sample01.py
temp  = int(input("Enter any number between 0 and 100 : "))

while(True):
    guess = int(input("Guess : "))
    if temp < guess :
        print("DOWN")
    elif temp > guess :
        print("UP")
    else:
        print("BINGO")
        break

print("Game is over")

